svd-rs/src/peripheral.rs

```diff
@@ -99,7 +99,7 @@ pub struct PeripheralBuilder {
     base_address: Option<u64>,
     default_register_properties: RegisterProperties,
     address_block: Option<Vec<AddressBlock>>,
-    interrupt: Vec<Interrupt>,
+    interrupt: Option<Vec<Interrupt>>,
```

Comment:
Why `Option<Vec<...>>`?

Reply:
Ah, enables modify_from to set empty registers

Reply:
I knew you'll ask this. This is because I use builders here https://github.com/burrbull/svdtools/tree/riir2 in 2 ways. For building structs and modifying. In such cases None means "do nothing" with interrupt where empty Vec means "delete all" interrupts.